packages/@lwc/style-compiler/src/selector-scoping/validate.ts

```diff
@@ -24,7 +24,7 @@ function validateSelectors(root: Root) {
             }
 
             // Ensure the selector doesn't use an unsupported selector.
-            if (UNSUPPORTED_SELECTORS.has(value)) {
+            if (!native && UNSUPPORTED_SELECTORS.has(value)) {
                 throw root.error(`Invalid usage of unsupported selector "${value}".`, {
```

Comment:
```suggestion
                throw root.error(`Invalid usage of unsupported selector "${value}". This selector is only supported in non-scoped CSS where the \`disableSyntheticShadowSupport\` flag is set.`, {
```

Reply:
I think this should cover it. Will need a prettier format and re-run test fixtures. :)

Reply:
Done! Thanks!